tdms/src/openandorder.cpp

```diff
@@ -60,8 +60,13 @@ int main(int nargs, char *argv[]){
     openandorder(args.grid_filename(), (char **)matrixnames_gridfile, matrixptrs, 1);
   }
 
+  // decide which derivative method to use (PSTD or FDTD)
+  DerivativeMethod method = PseudoSpectral; // default
```

Comment:
```suggestion
  SolverMethod method = PseudoSpectral; // default
```
I think `SolverMethod` is a better description, since my understanding is that psuedo-spectral and finite-difference refer to the method employed when time-evolving the system. I (think) I've caught all the instances of `DerivativeMethod` in my review, but I haven't touched `iterator.cpp` so a find+replace is be needed there. 

Tagging @prmunro in case I'm misunderstanding. Finite-difference timestepping happens in the the time-domain, based off the usual difference equation $$f(t_{i+1}) = \Delta_t f'(t_{i}) + f(t_{i}).$$
Psuedo-spectral methods do the timestepping in frequency (Fourier) space instead, since differentiation in real/time corresponds to multiplication in Fourier/frequency space:
$$\frac{\mathrm{d}f}{\mathrm{d}t}(t+\Delta_{t}) = \mathcal{F}^{-1}[D_t\mathcal{F}[f(t)]],$$
where the $D_t$ is the operator that `init_diff_shift_op` computes.

Reply:
`DerivativeMethod`  is my shoddy naming – apologies